llm/llama-2/chatbot-hf.yaml

```
@@ -19,6 +19,10 @@ setup: |
   pip install git+https://github.com/lm-sys/FastChat.git
   # Need the latest transformers to support 70B model
   pip install git+https://github.com/huggingface/transformers.git
+  pip install torch==1.12.1+cu113 --extra-index-url https://download.pytorch.org/whl/cu113
+  pip install fschat
+  pip install accelerate protobuf
+  pip install gradio==3.48.0
```

The following line alone seems working. Also, L19-L21 can be removed.
```suggestion
  pip install "fschat[model_worker,webui]"
```

---

@Michaelvll Yes, it seems to be working for me with this dependency, though, I updated it accordingly.